src/main/java/miku/command/UnmarkCommand.java

```diff
@@ -0,0 +1,35 @@
+package miku.command;
+
+import miku.utility.Storage;
+import miku.utility.TaskList;
+import miku.utility.UI;
+
+/**
+ * Unmarks a specified task.
+ */
+public class UnmarkCommand extends Command {
+    int index;
```

Comment:
I think you forgot to add a private modifier to this field, but other than that the rest looks great!